ConcernsCaseWork/ConcernsCaseWork/Security/IRbacManager.cs

```diff
@@ -6,9 +6,9 @@ namespace ConcernsCaseWork.Security
 {
 	public interface IRbacManager
 	{
-		IList<string> GetDefaultUsers();
-		Task<IDictionary<string, RoleClaimWrapper>> GetUsersRoles();
```

Comment:
I think we use Async suffixes on async method names in Concerns (although not in TRAMS API)

Reply:
I have definitely got cases where an async suffix isn't applied.
Lets talk about it and maybe we build a rule into a .editor config file